src/ocamlorg_web/lib/handler.ml

```diff
@@ -396,20 +386,27 @@ let package_docs t req =
   match version_opt with
   | None -> not_found req
   | Some version ->
-      let version = Ocamlorg_package.Version.to_string version in
-      let target = Ocamlorg_frontend.Url.package_doc package version in
+        let version_string = if Ocamlorg_package.is_latest_version t name version then 
+         "latest" else
+          Ocamlorg_package.Version.to_string version in
+      let target = Ocamlorg_frontend.Url.package_doc package version_string ~is_latest_url:false in
       Dream.redirect req target
```

Comment:
Sorry I overlooked this last time. See the comment above the function `package_docs`. It says

```
(** Redirect any URL with suffix /p/PACKAGE/docs to the latest documentation for
    PACKAGE. *)
```

So I would expect that we can simplify this, similar to how `Url.package` was simplified.

Though, I think, this function and the corresponding route could probably be removed altogether. :thinking: Oh well, that would be the topic for another PR.

Reply:
I think I am misunderstanding your comment. Do you mean to take out the `version_string` variable and just get the latest package with the Url.package function? Cause if I do that, then we dont use the version gotten from the match case

Reply:
Ahh, sorry, I didn't mean `Url.package`. I meant the `package` handler function just above this function. It is much shorter and much simpler because we always redirect to the `latest`-URL.